Site/HowToContribute.md

```diff
@@ -10,20 +10,16 @@ Do you want to help the community work toward a long-term goal of advancing scie
 
 The first step when it comes to contributing to BSSw is checking whether your content a good fit.  Please see [What To Contribute](./WhatToContribute.md) for information on scope.  If you are still unsure, please [contact us](https://bssw.io/pages/questions-about-contributing-to-better-scientific-software).
 
-PLEASE NOTE that at this stage, you are expected to only submit a short abstract of your article. Once reviewed and if accepted, you will be contacted to submit the complete version. Once you are ready to contribute, please use one of the below approaches to submit your abstract.  
-
-### Submission via Google form
-1. If you are not familiar with GitHub, you can submit your abstract using the [Google form](https://goo.gl/forms/IJyJKbI5OK9Lzcna2).
+PLEASE NOTE that at this stage, you are expected to only submit a short abstract of your article. Once reviewed and if accepted, you will be contacted to submit the complete version. Once you are ready to contribute and if you are familiar with GitHub, please do use the below approach to submit your abstract. If you are not familiar with GitHub, please contact us and we can help you.
```

Comment:
- Is this page really about learning how to contribute _content_ or is it about contributing an *idea* for content? I think it is the latter. But, text at top says its about contributing content itself.
- If this page is about contributing content itself (not just an idea), why do they need to submit an *abstract*? Why do we need to *review* it?
- Do we really mean to have people submit an *abstract* here? Or, do we mean to have them submit an *idea*? The word *abstract* connotes more fomalism than I think we need and enough formalism that it could be seen as a bigger hurdle to prospective submitters...especially if the goal is to gather ideas.
- All the instructions below ultimately go through our GitHub issues. So, IMHO, that is clear evidence that what we want this page to do is encourage people to submit *ideas* for content. Those will get injected into our regular content meetings and get voted, triaged, etc. (or that was the plan anyway).
- When we decide the idea is a good one, then instructions for how to contribute come in to play. But, IMHO, not before then.